html_classes_obfuscator/html_classes_obfuscator.py

```diff
@@ -46,15 +46,13 @@ def html_classes_obfuscator(htmlfiles = [], cssfiles = [], jsfiles = [], generat
 
                 old_no_quote = "class=" + classes_groups[i]
                 old_with_quote = 'class="' + classes_groups[i] + '"'
-                replace_by = 'class="' + obfuscate_classes_groups[i] + '"'
+                replace_by = 'class=' + obfuscate_classes_groups[i] + ''
```

Comment:
Thank you for your pull request !

The first commit you made did remove all the class quotes.

It works when we have only one class in the class attribute, like:

```html
<div class="hello"></div>

// and your PR generates:
<div class=aa></div>
```

But we can imagine having multiple classes too like:

```html
<div class="hello world"></div>

// and your PR generates:
<div class=aa bb></div>
```

which it is a wrong html output.

---

We may test if `obfuscate_classes_groups[i]` contains multiple classes inside:
- if yes, we keep the quotes
- if no, we remove them

Or doing something like that 😊

Reply:
Yes, it is, to generate valid HTML code

Reply:
Ok, I think I fixed it